src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -70,6 +80,16 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
     reset({ reviewNotes: '' });
   };
 
+  const handleShowWarning = (data: FormInput) => {
+    const scr = cr as StandardChangeRequest;
+    if (scr.proposedSolutions.find((ps) => ps.id === data.psId)!.timelineImpact > 0) {
```

Comment:
`!`  can't be a good idea here. what if there are no proposed solutions

Reply:
All Standard Change Requests have at least one proposed solution